librz/bin/dwarf/value.inc

```diff
@@ -0,0 +1,185 @@
+static inline int leading_zeros(uint64_t x) {
+#if defined(__GNUC__) || defined(__clang__)
+	return __builtin_clzll(x);
```

Comment:
The presence of it should be checked in meson.build, see the example of `__builtin_bswap16`